app/views/shared/_header.html.erb

```diff
@@ -30,7 +30,7 @@
             <ul class="dropdown-menu">
               <% Environment.all.each do |env| %>
                 <li>
-                  <%= link_to dashboard_path(env) do %>
+                  <%= link_to dashboard_path(env.name) do %>
```

Comment:
basically I'll need `Permalinkable` to avoid fun things like `.` in the name and things that make it non-unique